snap/snaptest/snaptest.go

```diff
@@ -203,3 +203,47 @@ func MakeTestSnapWithFiles(c *check.C, snapYamlContent string, files [][]string)
 	}
 	return filepath.Join(snapSource, snapFilePath)
 }
+
+// RenameSlot renames gives an existing slot a new name.
+//
+// The new slot name cannot clash with an existing plug or slot and must
+// be a valid slot name.
+func RenameSlot(snapInfo *snap.Info, oldName, newName string) error {
```

Comment:
You can drop it in `export_test.go` if it's just for testing.

Reply:
It is for testing but _elsewhere_. I will use it in the interface repository tests.

Reply:
Heh, completely missed that it's in `snaptest`.